java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -246,6 +252,15 @@ public static Object[] transactionTestResult() {
             2L, // zadd(zSetKey2, Map.of("one", 1.0, "two", 2.0))
             new String[] {"one", "two"}, // zdiff(new String[] {zSetKey2, key8})
             Map.of("one", 1.0, "two", 2.0), // zdiffWithScores(new String[] {zSetKey2, key8})
+            new String[] {"one", "two"}, // zunion(new KeyArray(new String[] {zSetKey2, key8}))
+            new String[] {
+                "one", "two"
+            }, // zunion(new KeyArray(new String[] {zSetKey2, key8}), Aggregate.MAX);
```

Comment:
you can shorten comments to fit this into a single line
```suggestion
            new String[] { "one", "two" }, // zunion(new KeyArray([zSetKey2, key8]), Agg.MAX);
```